wire-ios/Wire-iOS/Sources/Components/Buttons/IconButton.swift

```diff
@@ -182,7 +182,6 @@ class IconButton: ButtonWithLargerHitArea {
 
     func setBackgroundImageColor(_ color: UIColor,
                                  for state: UIControl.State) {
-        let color = color.resolvedColor(with: traitCollection)
```

Comment:
So is this method invoked automatically when the theme changes?

Reply:
@johnxnguyen Exactly.

Reply:
I've also tested it with an iOS 15, 16 and 17 simulator and had no issues!

Reply:
I'm curious where it's being invoked from? Perhaps we can look at the debugger together?